DBZ-9187 Removing zookeeper and using Kafka KRaft (#6585)
debezium/debezium: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-kafka.adoc

[source,shell,options="nowrap",subs="+attributes"]
----
$ docker run -it --rm -p 9092:9092 \
  --name kafka --hostname kafka \
  -e CLUSTER_ID=<YOUR_UNIQUE_CLUSTER_IDENTIFIER> \
  -e NODE_ID=1 \
  -e NODE_ROLE=combined \
  -e KAFKA_CONTROLLER_QUORUM_VOTERS=1@kafka:9093 \
  -e KAFKA_LISTENERS=PLAINTEXT://kafka:9092,CONTROLLER://kafka:9093 \
  -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092 \
  quay.io/debezium/kafka:{debezium-docker-label}
----

`-it`:: The container is interactive,
which means the terminal's standard input and output are attached to the container.
`--rm`:: The container will be removed when it is stopped.
`--name kafka`:: The name of the container.

> `--hostname kafka`:: To ensure that the container is identified as kafka, for a correct listener function.

Reviewer: ```suggestion
`--hostname kafka`:: Label ensures that the correct Kafka listener is paired to the container.
```

`-p 9092:9092`:: Maps port `9092` in the container to the same port on the Docker host so that applications outside of the container can communicate with Kafka.

> `-e CLUSTER_ID=<YOUR_UNIQUE_CLUSTER_IDENTIFIER>`:: Unique cluster identifier used in your cluster. All the nodes inside the cluster has to have the same value.

Reviewer: ```suggestion
`-e CLUSTER_ID=<YOUR_UNIQUE_CLUSTER_IDENTIFIER>`:: Unique identifier in the cluster.
All the nodes within the cluster must have the same cluster ID.
```

> `-e NODE_ID=1`:: Node identenfier used in Raft protocol. Must be unique inside the cluster.

Reviewer: ```suggestion
`-e NODE_ID=1`:: Node identifier used in Raft protocol. 
Must be unique within the cluster.
```

> `-e NODE_ROLE=combined`:: Node role inside the cluster. Can be a controller, broker or combined,

Reviewer: ```suggestion
`-e NODE_ROLE=combined`:: Node role within the cluster. 
Can be a controller, broker, or combined,
```

> `-e KAFKA_CONTROLLER_QUORUM_VOTERS=1@kafka:9093`:: Specifies which nodes act as votwers in the Kafka controller quorum.

Reviewer: ```suggestion
`-e KAFKA_CONTROLLER_QUORUM_VOTERS=1@kafka:9093`:: Specifies the nodes act that act as voters in the Kafka controller quorum.
```
Does this key support multiple values, or does it apply only to a single node?

> `-e KAFKA_LISTENERS=PLAINTEXT://kafka:9092,CONTROLLER://kafka:9093`:: Defines rthe internal endpoints and protocol for Kafka listeners.

Reviewer: ```suggestion
`-e KAFKA_LISTENERS=PLAINTEXT://kafka:9092,CONTROLLER://kafka:9093`:: Defines the internal endpoints and protocol for Kafka listeners.
```
